src/mscorlib/src/System/Tuple.cs

```diff
@@ -118,7 +118,7 @@ public Tuple(T1 item1)
 
         public override Boolean Equals(Object obj)
         {
-            return ((IStructuralEquatable)this).Equals(obj, EqualityComparer<Object>.Default);
+            return ((IStructuralEquatable)this).Equals(obj, null);
```

Comment:
>null [](start = 60, length = 4)

Why now pass `null`? I'm tempted to leave as before. Maybe BCL reviewers can comment.